contrib/charts/dragonfly/ci/priorityclassname-values.golden.yaml

```diff
@@ -101,3 +92,12 @@ spec:
           resources:
             limits: {}
             requests: {}
+---
```

Comment:
So, Sorry for the late reply but I tried pushing the required change for the CI to pass but because the fork seems to be in an org, I get the following error:

```
dragonfly/contrib/charts/dragonfly  🍣 helm-add-host-network 📦 📝 ×1🛤️  ×13via 🐹 v1.21.6 via ⎈ v3.15.4 on ☁️  (us-east-1) on ☁️  tarun@dragonflydb.io 
🕙 13:18:53 ❯ git push lablabs helm-add-host-network                                  
remote: Permission to lablabs/dragonfly-operator.git denied to Pothulapati.
fatal: unable to access 'https://github.com/lablabs/dragonfly-operator.git/': The requested URL returned error: 403
```

So, The issue is with the local helm version. Bump it to `v3.15.4`, and run the test update command, you should see some changes. Once they are pushed, CI should pass. Thanks! :)

Reply:
@Pothulapati hey no problem. I've updated mu helm version to v3.15.4 and now I see a change. I have comited the change